lib/sorbet-rails/model_utils.rb

```diff
@@ -73,9 +83,10 @@ def exists_class_method?(method_name)
       root: Parlour::RbiGenerator::Namespace,
       method_name: String,
       parameters: T.nilable(T::Array[::Parlour::RbiGenerator::Parameter]),
+      standard_query_method: T::Boolean,
     ).void
   }
-  def add_relation_query_method(root, method_name, parameters: nil)
+  def add_relation_query_method(root, method_name, parameters: nil, standard_query_method: false)
```

Comment:
I think it's not obvious when to pass `standard_query_method` and when to use the default value. Can you add a comment about it? It seems it should be true for "Rails" provided query methods, and examples of non-standard methods are named scopes. Do I understand it correctly?